zulip_bots/zulip_bots/terminal.py

```diff
@@ -56,18 +56,22 @@ def main():
 
     sender_email = 'foo_sender@zulip.com'
 
-    while True:
-        content = input('Enter your message: ')
+    try:
+        while True:
+            content = input('Enter your message: ')
 
-        message = dict(
-            content=content,
-            sender_email=sender_email,
-            display_recipient=sender_email,
-        )
-        message_handler.handle_message(
-            message=message,
-            bot_handler=bot_handler,
-        )
+            message = dict(
+                content=content,
+                sender_email=sender_email,
+                display_recipient=sender_email,
+            )
+            message_handler.handle_message(
+                message=message,
+                bot_handler=bot_handler,
+            )
+    except:
```

Comment:
The build is failing for exactly what I would request as a change, namely a more-specific `except` statement here :)